src/js/index.js

```diff
@@ -0,0 +1,121 @@
+import Books from './modules/books_class_module.js';
+import now from './modules/luxon.js';
+
+const date = document.querySelector('.date');
+date.textContent = now;
+
+const form = document.querySelector('#form');
+const inputs = form.querySelectorAll('input');
+const bookContainer = document.querySelector('ul.book-list');
+const booksListNav = document.querySelector('.nav-item.vl.nav-book-list');
+const booksFormNav = document.querySelector('.nav-item.vl.nav-add-books');
+const contactsNav = document.querySelector('.nav-item.nav-contacts');
+const message = document.querySelector('p.message');
+const bookFormSection = document.getElementById('book-form');
+const contacts = document.getElementById('contact');
+const bookContainerSection = document.getElementById('added-books');
+let TitleInputField;
+let AuthorInputField;
+let tab = 'book-list';
+
+const booksClass = new Books();
+const updateStorage = () => {
+  const stringfiedBL = JSON.stringify(booksClass.getBooks());
+  localStorage.setItem('booklist', stringfiedBL);
+};
+const renderBooks = () => {
+  const Books = booksClass.getBooks();
+  let li = '';
+  if (Books.length) {
+    Books.forEach((book, index) => {
+      li += `<li class="book-item ${index % 2 ? '' : 'bg-black'}">
+       <div class="book-title">"${book.title}" by ${book.author}</div>
+       <button class="btn-remove" id=${book.id}>remove</button>
+   </li>`;
+    });
+  } else {
+    li += '<li class="center">Book list is empty!</li>';
+  }
+
+  bookContainer.innerHTML = li;
+  const removeBtn = document.querySelectorAll('.btn-remove');
+  removeBtn.forEach((button) => button.addEventListener('click', () => {
+    booksClass.removeBook(button.id);
+    updateStorage();
+    renderBooks();
+  }));
+};
+const addBooks = (title, author, id) => {
+  booksClass.addBooks(title, author, id);
+  renderBooks();
+  updateStorage();
+  message.textContent = 'Book added';
+  TitleInputField.value = '';
+  AuthorInputField.value = '';
+};
+function updateUserInterface() {
```

Comment:
@Cathella thank you for your comment, I have changed that and will submit another code review.
Regarding `luxon` here https://github.com/Elerqsousy/awesome-books-es6-version/blob/6d67195e2b53de06519a8f7917232965a36f9e7b/src/js/modules/luxon.js#L2

I am not comfortable using the node_module path to access luxon. Can I use require or import with node.js and then request in the code review to install and use a bundler such as Parcel which is installed in the project package, as it will only work on the server-side and not the client-side?
such as this line here: `import { DateTime } from 'luxon';`

Thank you in advance for your support and time.

Reply:
Hello @Elerqsousy, I'm not so sure but I think you are not obliged to use this way of importing from that long path, you can try to use this method above, the one importing from luxon directly, and see if it will work

Reply:
@moise10r Thank you very much for replying to this thread.
I have tried it before, and it works.